From 6501a197ae0f3d84c7732e1e6a948d973491972c Mon Sep 17 00:00:00 2001
From: Joan Bruguera <joanbrugueram@gmail.com>
Date: Fri, 25 Mar 2022 20:00:36 +0100
Subject: [PATCH] Fix syntax error on generated configure script

Regression introduced in f8e1ccdb5b54604a8827480e767b41c9f164ef8c

It appears that (at least on my system) when `AX_FIND_INSTALLATION` is called
with an empty array on `$6=list-of-optional-headers`, an empty if body is
generated and the configure script fails with:

    ./configure: line 11487: syntax error near unexpected token `fi'

Let's put something in the if body to work around it
---
 config/macros.m4 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/config/macros.m4 b/config/macros.m4
index d009f859..9afc11c8 100644
--- a/config/macros.m4
+++ b/config/macros.m4
@@ -212,6 +212,7 @@ dnl AX_FLAGS_SAVE()
     dnl Test for optional headers
     if test -n "$6"; then
       AC_CHECK_HEADERS($6)
+      true
     fi
   fi
 
-- 
2.35.1

